### DISCUSSION TITLE
12 Step Meeting Wordpress Plugin - and PDF generator

### DISCUSSION THREAD

**Discussion by jlabenberg** [code4recovery/12-step-meeting-list#1114] [Q&A]
I am a newbie to website building and wordpress.  I inherited an antiquated website that was created with HTML about 20 years ago and I am tasked with developing a new website using wordpress.

One of the most important parts of the new website is the meeting list and I am using 12 step meeting plugin - which is fantastic!  I have successfully installed the plugin and have entered all the relevant data.  I also want the user to be able to generate a PDF of the meeting schedule - using the PDF generator in the plugin and I want to have a link on the home page for the pdf meeting list but I cannot figure out where (the url) for the pdf meeting list is.

can someone help me with this?

site = d48-wp-2023.district48aa.org

**Comment by joshreisner** [comment]
hi @jlabenberg - our PDF generation system is accessed from the import / export page of the plugin. it's designed for site admins to generate a printed list, not for individual users to generate on-the-fly PDFs

many offices print new lists periodically, for example monthly - one approach you can take is to upload that PDF as a static asset to WordPress and link to it from your site navigation, or from a special page, eg "latest printed list (apr 1)" 

we recommend against putting the link on the meeting list page itself, as some offices do. this prioritizes the edge case of wanting to print a list over the very common use case of simply needing to find a meeting
